modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/resources.jsp

```diff
@@ -35,6 +35,34 @@ String inlineEditSaveURL = GetterUtil.getString((String)request.getAttribute(CKE
 	long javaScriptLastModified = PortalWebResourcesUtil.getLastModified(PortalWebResourceConstants.RESOURCE_TYPE_EDITOR_CKEDITOR);
 	%>
 
+	<script data-senna-track="temporary" type="text/javascript">
+		window.CKEDITOR_GETURL = function (resource) {
+			var languageParameter = '&languageId=' + themeDisplay.getLanguageId();
+
+			// If this is not a full or absolute path.
+
+			if (resource.indexOf(':/') == -1 && resource.indexOf('/') !== 0) {
```

Comment:
Please be consistent with triple equals. Here you're using double (`==`) and triple (`!==`) on the same line. You're using double below on line 52. Until we get our `eqeqeq` lint rule active, just use triple equals/not-equals all the time.

Independently of that, we could probably tighten up the conditional a bit here. I gather that with your check for ":/" you are trying to detect an scheme, but ":/" could legitimately appear elsewhere in a URL. (Conversely, a schemaless URL — eg. `//example.com/hey` — is an "absolute" URL that doesn't have a ":/" in it, but your `startsWith()` check *will* handle that one.)